api/libs/scenarios-planning-unit/src/domain/lock-planning-units.ts

```diff
@@ -0,0 +1,11 @@
+import { differenceWith } from 'lodash';
+
+export const lockPlanningUnits = (
```

Comment:
I suggest to clarify the function name - this function is not actually locking units nor even returning a difference of the lockable units, but checking if the units to lock are a non-strict subset of the available ones.

Reply:
@hotzevzl fixed in https://github.com/Vizzuality/marxan-cloud/pull/224/commits/8b9c4a7bbed5d99bc818bedd331199dc29415f8e